### ISSUE TITLE
Inconsistency in row names in netsim $stats$nwstats output

### ISSUE THREAD

**Issue by sgoodreau** [statnet/EpiModel#309]
Minor issue:  in the object returned by `netsim`, the row names in `$stats$nwstats` are:

1
[2,]
[3,]
etc.  

For example:
```
library(EpiModel)
mynet <- network.initialize(100, directed = F)
mymodel <- netest(mynet, formation = ~edges, target.stats = 40,
                            coef.diss = dissolution_coefs(~offset(edges), 100, 0))
myparams <- param.net(inf.prob = 0.01, act.rate = 0.1,
                                a.rate = 0, ds.rate = 0, di.rate = 0)
myinit <- init.net(i.num = 10)
mycontrol <- control.net(type='SI', nsteps = 10, nsims = 1, verbose=TRUE)
mysim <- netsim(mymodel, myparams,myinit, mycontrol)
mysim$stats$nwstats$sim1
class(mysim$stats$nwstats$sim1)
rownames(mysim$stats$nwstats$sim1)
```
The behavior remains even if there is >1 column in the data.frame (i.e. there is >1 statistic in the model) or if nsims>1.
I am using EpiModel 1.7.2:

```
> sessionInfo(
+ )
R version 3.5.2 (2018-12-20)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows Server >= 2012 x64 (build 9200)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252 
[2] LC_CTYPE=English_United States.1252   
[3] LC_MONETARY=English_United States.1252
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] EpiModel_1.7.2       tergm_3.5.2          ergm_3.9.4          
[4] networkDynamic_0.9.0 network_1.13.0.1     deSolve_1.21        

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.0           RColorBrewer_1.1-2   plyr_1.8.4          
 [4] pillar_1.3.1         compiler_3.5.2       DEoptimR_1.0-8      
 [7] bindr_0.1.1          iterators_1.0.10     tools_3.5.2         
[10] gtable_0.2.0         tibble_1.4.2         nlme_3.1-137        
[13] lattice_0.20-38      pkgconfig_2.0.2      rlang_0.3.0.1       
[16] foreach_1.4.4        Matrix_1.2-15        rstudioapi_0.8      
[19] yaml_2.2.0           parallel_3.5.2       xfun_0.4            
[22] bindrcpp_0.2.2       coda_0.19-2          dplyr_0.7.8         
[25] knitr_1.21           trust_0.1-7          grid_3.5.2          
[28] tidyselect_0.2.5     glue_1.3.0           robustbase_0.93-3   
[31] R6_2.3.0             purrr_0.2.5          ggplot2_3.1.0       
[34] magrittr_1.5         codetools_0.2-16     scales_1.0.0        
[37] MASS_7.3-51.1        assertthat_0.2.0     ape_5.2             
[40] colorspace_1.3-2     lpSolve_5.6.13       doParallel_1.0.14   
[43] lazyeval_0.2.1       munsell_0.5.0        statnet.common_4.1.4
[46] crayon_1.3.4         
```

**Comment by smjenness** [comment]
The preferred way to extract network stats from both `netdx` and `netsim` objects is the `get_nwstats` function, which tidies up the output. See the help page for usage and examples.

**Comment by sgoodreau** [comment]
True, although you've undoubtedly noticed through the years that I'm someone who likes to great straight into the object itself and explore it directly, and I don't think I'm alone in that among the likely user base.  My philosophy is: if we're returning an object from an exported function, it really is OK for the user to interact with it how they want through any reasonable means.

This particular issue really is no big deal either way - just thought you might want to know of the oddity, if you didn't already.

**Comment by smjenness** [comment]
Yes, that's fine. Please submit a PR with your proposed fix. Thank you!

**Comment by smjenness** [comment]
Closed by #310
